src/_posts/platform/getting-started/2000-01-01-setup-ssh-linux.md

```diff
@@ -1,6 +1,6 @@
 ---
 title: Setup SSH on Linux
-modified_at: 2021-10-25 00:00:00
+modified_at: 2022-01-06 00:00:00
```

Comment:
Is it wanted to update the date here?

Reply:
Yep it was forgotten during last update. I checked when this file was last modified and updated it.